css/style.css

```diff
@@ -0,0 +1,21 @@
+#main{
+    margin: 4%;
+}
+#img-div {
+    width: 100%;
+    position: relative;
+    /* Center image*/
+    display: flex;
+    justify-content: center;
+}
+
+#image {
+    display: block;
+    max-width: 30%;
+    height: auto;
+    position: absolute;
+}
+
+#img-caption {
+  position: absolute;
+}
```

Comment:
For readability please make sure to use two spaces for indentation. 
Do not use the tab space or if your code editor allows you to edit the number of spaces the tab gives, then that will be fine.

Reply:
- [Optional ] please be polite 👍